openedx/core/djangoapps/content/block_structure/tests/test_signals.py

```diff
@@ -22,25 +21,21 @@ def setUp(self):
         self.course = CourseFactory.create()
         self.course_usage_key = self.store.make_course_usage_key(self.course.id)
 
-    def test_course_publish(self):
```

Comment:
Why were these tests deleted?  You can import `get_course_blocks` from the lib directory and continue to keep these tests.